src/Features/Core/Portable/Diagnostics/EngineV1/DiagnosticIncrementalAnalyzer.StateManager.cs

```diff
@@ -116,6 +132,38 @@ public void RemoveStateSet(ProjectId projectId)
                 _projectStates.RemoveStateSet(projectId);
             }
 
+            private List<StateSet> CreateBuildOnlyProjectStateSet(Project project)
+            {
+                var referenceIdentities = project.AnalyzerReferences.Select(r => _analyzerManager.GetAnalyzerReferenceIdentity(r)).ToSet();
+                var stateSetMap = GetStateSets(project).ToDictionary(s => s.Analyzer, s => s);
+
+                var stateSets = new List<StateSet>();
+                var analyzerMap = _analyzerManager.GetHostDiagnosticAnalyzersPerReference(project.Language);
```

Comment:
we are not, this code is to remove stateset that are only for host analyzers. this code is to see whether there is analyzer that exist both in project and host, and if they do, we include it here.